drivers/hap_report/app/models/hap_report/queries.rb

```diff
@@ -0,0 +1,148 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HapReport::Queries
+  extend ActiveSupport::Concern
+  include  ArelHelper
+
+  included do
+    def a_t
+      HapReport::HapClient.arel_table
+    end
+
+    # Project types
+    def bridge_housing
+      lit('2').eq(any(a_t[:project_types]))
+    end
+
+    def case_management
+      lit('6').eq(any(a_t[:project_types]))
+    end
+
+    def rental_assistance
+      lit('12').eq(any(a_t[:project_types]))
+    end
+
+    def emergency_shelter
+      lit('1').eq(any(a_t[:project_types]))
+    end
+
+    def innovative
+      lit('12').eq(any(a_t[:project_types]))
+    end
+
+    def total
+      Arel.sql('1=1')
+    end
+
+    # Populations
+    def adults
+      a_t[:age].gteq(18).or(a_t[:emancipated].eq(true))
+    end
+
+    def children
+      a_t[:age].lt(18).and(a_t[:emancipated].eq(false))
+    end
+
+    def households
+      @households ||=
+        {}.tap do |hash|
+          report_client_scope.each do |client|
+            hap_client = client.universe_membership
+            hap_client.household_ids.each do |h_id|
+              hash[h_id] ||= []
+              hash[h_id] << {
+                client_id: hap_client.id,
+                head: hap_client.head_of_household_for.include?(h_id),
+                age: hap_client.age,
+              }
+            end
+          end
+        end
+    end
+
+    def families_with_children
+      @families_with_children ||= households.
+        select { |_, v| v.any? { |client| client[:age] < 18 } }.
```

Comment:
Does it matter that this doesn't check to confirm the household also contains adults?